fe/src/shared/components/OrderItemStatusChanges/OrderItemStatusChanges.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+    Table, TableBody, TableCell, TableContainer,
+    TableHead, TableRow, Paper, NoSsr
+} from "@mui/material";
+import moment from "moment";
+
+const OrderItemStatusChanges = (props) => {
+    const { labels, statusChanges } = props;
+    
+    return (
+        props.statusChanges &&
+            <div className="order-history mt-5 p-5">
+                <h3 className="title is-4">{labels.title}</h3>
```

Comment:
**suggestion:** this line throws an error because labels are null for existing order items